package.json

```diff
@@ -26,6 +26,7 @@
     "es6-promise": "^2",
     "eslint": "^1.9.0",
     "eslint-plugin-no-unsafe-innerhtml": "^1.0.14",
+    "eslint-plugin-no-wildcard-postmessage": "0.1.2",
```

Comment:
I should have been more verbose in the review request. I'm very much interested in your thoughts about the plugin. Did you give it a full review or are these just drive-by comments?

I'd prefer this add-on being regarded mature enough before bringing this in. I'll file issues and work on the things you have found so far.